### PR TITLE
Add release process document

### PR DESCRIPTION
**Pull request by bryteise**
Signed-off-by: William Douglas <william.douglas@intel.com>

### CODE REVIEW

File: how-to-make-a-release [hunk 1]
```diff
@@ -0,0 +1,4 @@
+1) Update VERSION file and commit
+2) Tag new release with "v${VERSION}"
+3) Run "/scripts/git-archive-all.sh --format tar --prefix clr-boot-manager-${VERSION}/ -t v${VERSION} clr-boot-manager-${VERSION}.tar && xz clr-boot-manager-${VERSION}.tar"
```
Comment (bwarden) [inline how-to-make-a-release:3]:
```suggestion
3) Run "./scripts/git-archive-all.sh --format tar --prefix clr-boot-manager-${VERSION}/ -t v${VERSION} clr-boot-manager-${VERSION}.tar && xz clr-boot-manager-${VERSION}.tar"
```

Comment (bwarden) [reply how-to-make-a-release:3]: Or does `./scripts/mkrelease.sh` just need to be tweaked a bit to accomplish all of these steps instead?

Comment (bryteise) [reply how-to-make-a-release:3]: mkrelease.sh will need to be poked a little. I don't use it since I don't sign the archives.

Comment (bryteise) [reply how-to-make-a-release]: Updated

**Review by bwarden — APPROVED** [review]

**Pull request merged**